docs/content/direct/core-chart.md

```diff
@@ -190,89 +190,115 @@ the `kflex` CLI and one not.
 
 1. Using `kflex` CLI
 
-    The following commands will add a context, named after the given
-    control plane, to your current kubeconfig file and make that the
-    current context. The deletion is to remove an older vintage if it
-    is present.

```

Comment:
Why remove the indenting? It is here for a reason: so that the numbered list structure is recognized properly by the mkdocs rendering.

Reply:
ok, I did not know it was necessary

Reply:
Yes, we also need to add the indenting elsewhere. I am happy to leave that to a distinct PR.

Reply:
I still see a lot of indenting being removed by this PR.

Reply:
Mike I think it is the opposite, I restored the indents that I removed and added matching indents in the other sections

Reply:
Oops, you're right!